app/Http/Controllers/UstensilsController.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kami\Cocktail\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Http\JsonResponse;
+use Kami\Cocktail\Http\Requests\UstensilRequest;
+use Kami\Cocktail\Http\Resources\UstensilResource;
+use Kami\Cocktail\Models\Ustensil;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class UstensilsController extends Controller
```

Comment:
Is there any difference between spellings `ustensils` and `utensils`. https://en.wikipedia.org/wiki/Kitchen_utensil